docs/auth.md

```diff
@@ -0,0 +1,28 @@
+# Auth & Cookies
+
```

Comment:
@aldeed @ticean @rosshadden these proposed changes to how we use cookies need a few more eyes to review. I've written these readme to list the changes and will like comments from you all.

Reply:
Looks good to me @impactmass.